go.mod

```diff
@@ -1,41 +1,25 @@
 module sigs.k8s.io/cluster-api-addon-provider-helm
 
-go 1.20
-
-replace (
-	github.com/google/cel-go => github.com/google/cel-go v0.12.6
-	github.com/google/gnostic => github.com/google/gnostic v0.6.9
-	k8s.io/api => k8s.io/api v0.27.3
-	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.27.3
-	k8s.io/apimachinery => k8s.io/apimachinery v0.27.3
-	k8s.io/apiserver => k8s.io/apiserver v0.27.3
-	k8s.io/cli-runtime => k8s.io/cli-runtime v0.27.3
-	k8s.io/client-go => k8s.io/client-go v0.27.3
-	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.27.2
-	k8s.io/component-base => k8s.io/component-base v0.27.3
-	k8s.io/kube-openapi => k8s.io/kube-openapi v0.0.0-20230501164219-8b0f38b5fd1f
-	k8s.io/kubectl => k8s.io/kubectl v0.27.3
-	sigs.k8s.io/controller-runtime => sigs.k8s.io/controller-runtime v0.15.3
-	sigs.k8s.io/kustomize/kyaml => sigs.k8s.io/kustomize/kyaml v0.14.1
-)
+go 1.21
 
 require (
 	github.com/Masterminds/sprig v2.22.0+incompatible
 	github.com/google/go-cmp v0.6.0
-	github.com/onsi/ginkgo/v2 v2.13.1
+	github.com/onsi/ginkgo/v2 v2.14.0
 	github.com/onsi/gomega v1.30.0
 	github.com/pkg/errors v0.9.1
-	go.uber.org/mock v0.3.0
+	github.com/spf13/pflag v1.0.5
+	go.uber.org/mock v0.2.0
 	gopkg.in/yaml.v2 v2.4.0
-	helm.sh/helm/v3 v3.13.2
-	k8s.io/api v0.28.2
-	k8s.io/apimachinery v0.28.2
-	k8s.io/cli-runtime v0.28.2
-	k8s.io/client-go v0.28.2
-	k8s.io/klog/v2 v2.110.1
-	k8s.io/utils v0.0.0-20231121161247-cf03d44ff3cf
-	sigs.k8s.io/cluster-api v1.5.3
-	sigs.k8s.io/controller-runtime v0.15.1
+	helm.sh/helm/v3 v3.13.3
+	k8s.io/api v0.28.4
+	k8s.io/apimachinery v0.28.4
+	k8s.io/cli-runtime v0.28.4
+	k8s.io/client-go v0.28.4
+	k8s.io/klog/v2 v2.100.1
+	k8s.io/utils v0.0.0-20240102154912-e7106e64919e
+	sigs.k8s.io/cluster-api v1.6.0
```

Comment:
Should we bump this up to [CAPI v1.6.1](https://github.com/kubernetes-sigs/cluster-api/releases/tag/v1.6.1) now?

Reply:
I'm in favor of leaving it for now and bumping it once we merge the e2e test framework PR so we don't need to manually verify that a dependency bump doesn't break anything.